poutyne/framework/callbacks/lr_scheduler.py

```diff
@@ -72,7 +80,7 @@ def f(self, *args, **kwargs):
         setattr(sys.modules[__name__], name, _new_cls)
 
 
-class ReduceLROnPlateau(Callback):
+class ReduceLROnPlateau(LRScheduler):
```

Comment:
I think everything in this class except on_epoch_end is already in _PyTorchLRSchedulerWrapper. We should be able to simplify the code more.

Reply:
Agreed. Fixed by 36ca7281613c652f39858466acd6871427752a10